hermeznetwork/bridge-ui
# Add missing API calls

Closes #31 .

### What does this PR does?

This PR adds support for the missing API endpoints:

- GET /claim-status
- GET /merkle-proof
- GET /claims

### How to test?

Run the `hermez-bridge` API in your machine, write this code in the `Home` view and check the console:

```ts
const { getClaims, getMerkleProof, getClaimStatus } = useBridgeContext();

useEffect(() => {
  void getClaims({ ethereumAddress: "0xabCcEd19d7f290B84608feC510bEe872CC8F5112" }).then(
    console.log
  );
  void getMerkleProof({ originNetwork: 0, depositCount: 2 }).then(console.log);
  void getClaimStatus({ originNetwork: 0, depositCount: 2 }).then(console.log);
}, []);
```

## Checklist

These are the criteria that every PR should meet, please check them off as you
review them:
